### PR TITLE
DR-3193 Point original AnVIL compact id prefixes to TDR

### PR DESCRIPTION
**Pull request by nmalfroy**
Since all files are aliased in TDR now, we should be good to go

### CODE REVIEW

**Review by okotsopoulos — APPROVED** [review]

File: service/src/main/resources/application.yml [hunk 1]
```diff
@@ -136,7 +135,7 @@ spring.config.activate.on-profile: prod
 drshub:
   compactIdHosts:
     "dg.4503": gen3.biodatacatalyst.nhlbi.nih.gov # biodatacatalyst
-    "dg.anv0": gen3.theanvil.io # old anvil
+    "dg.anv0": data.terra.bio # old anvil (now hosted on TDR)
```
Comment (pshapiro4broad) [inline service/src/main/resources/application.yml]: Really minor but comments like `now hosted` only make sense relative to the time when the commit occurred. I would leave out the `now`.
```suggestion
    "dg.anv0": data.terra.bio # old anvil (hosted on TDR)
```

**Review by pshapiro4broad — APPROVED** [review]
Looks OK although I think there are some lines in the `README.md` that also need to be updated, for example:

```
### Dev
| Provider            | Compact Id (CIB)  | Host URI                                   |
|---------------------|-------------------|--------------------------------------------|
| AnVIL (gen3 hosted) | dg.anv0           | staging.theanvil.io                        |
| AnVIL (TDR hosted)  | drs.anv0          | jade.datarepo-dev.broadinstitute.org       |
```

File: service/src/main/resources/application.yml (hunk 1 above)
Comment (nmalfroy) [reply service/src/main/resources/application.yml]: fair enough

**Review by Shakespeared — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -149,7 +149,7 @@ Note: there are a few tricky cases with the **compact IDs (CID)**:
 ### Dev
 | Provider            | Compact Id (CIB)  | Host URI                                   |
 |---------------------|-------------------|--------------------------------------------|
-| AnVIL (gen3 hosted) | dg.anv0           | staging.theanvil.io                        |
+| AnVIL (gen3 hosted) | dg.anv0           | jade.datarepo-dev.broadinstitute.org       |
```
Comment (Shakespeared) [inline README.md]: maybe also change this parenthetical to `(TDR hosted)` or remove it

**Review by Shakespeared — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (nmalfroy) [reply README.md]: ah good call!

**Pull request merged**